roles/openshift_sanitize_inventory/action_plugins/check_deprecated.py

```diff
@@ -0,0 +1,49 @@
+"""
+Ansible action plugin to check for usage of deprecated variables in Openshift Ansible inventory files.
+"""
+import sys
+import os
+import base64
+import traceback
+import errno
+import json
+from collections import defaultdict
+
+from ansible.plugins.action import ActionBase
+from ansible.module_utils.six import string_types
+
+try:
+    from __main__ import display
+except ImportError:
+    # pylint: disable=ungrouped-imports; this is the standard way how to import
+    # the default display object in Ansible action plugins.
+    from ansible.utils.display import Display
+    display = Display()
+
+class ActionModule(ActionBase):
+    """Action plugin to execute deprecated var checks."""
+
+    def run(self, tmp=None, task_vars=None):
+        result = super(ActionModule, self).run(tmp, task_vars)
```

Comment:
We don't need to capture the result if we're not going to use it; I think proper actions set items inside this dictionary/object and finally return it, but my example didn't.

Reply:
updated to use the proper usage of `result`